### DISCUSSION TITLE
22.04

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#206] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/DISTRHO/Cardinal/releases/tag/22.04'>22.04</a>.</em>

**Comment by kaiwok** [comment]
Hello,

This release is not working anymore on my old MBP running OS X El Capitan 10.11.6 (15G1011) and Reaper v6.53.
The last working release was 22.02 (VST2 version). 

Is it a bug or a feature please ;) ?

Thanks a lot for your help and thanks a lot for this wonderful project !

**Reply by falkTX** [reply]
not working how? reporting unsupported version?
the minimum version set for these binaries is 10.12

**Reply by kaiwok** [reply]
In 22.03 and 22.04 Reaper is not able anymore to load the plug-in :
![image](https://user-images.githubusercontent.com/1015325/163676442-d7eeeb7d-b460-4d71-aaf8-015a58d08157.png)

I also saw a big size difference between 22.02 and new vst files (big code optimization :) ) : 
![image](https://user-images.githubusercontent.com/1015325/163676615-8432e084-68c0-4354-8439-57534f0687ba.png)

As I am using an old and a new MBP I will use the release 22.02 on both ;) 
Thank you for your help.

**Reply by falkTX** [reply]
if you are on 10.11, that would be the reason for it, as I said 10.12 is the minimum supported version.
I can though add macos-intel builds which support older 10.8, will keep that in mind for future builds after the current ones finish

**Reply by kaiwok** [reply]
Maybe it's time for me to upgrade my hardware ^^
Btw thank you very much for your amazing work!

**Reply by falkTX** [reply]
btw, the latest builds generated from github actions now have a macos-intel target, which in theory works on systems older than 10.12

**Reply by kaiwok** [reply]
Thank you very much for the effort but unfortunately the macos-intel version does not work. Never mind, I'll stick with 22.02 ;)

**Reply by falkTX** [reply]
22.02 is never going to be updated though.
we can try to find more info for why it fails.. does macOS give any error if trying something from the terminal? (ie, start the host via terminal, try to load plugin, see what is printed)

**Comment by jaffasplaffa** [comment]
Hey :) 

Just installed Cardinal on my M1 Macbook Pro. I use it in Logic 10.7.1, in Native Arm mode, OSX 11.6.5.

If I open two instances of the Cardinal Synth, only one of the plays the right audio. If I open the editor of one of them, that one will sound okay and the other one sounds wrong. If I then open the editor for the other one, that patch sounds right, but the other one sounds wrong.

Does Cardinal not allow more than one instance of it in a DAW yet?

Thanks in advance!

Jakob

**Reply by falkTX** [reply]
> Does Cardinal not allow more than one instance of it in a DAW yet?

Yes, nothing in Cardinal is special for this.

Might be an issue with a particular module, try with simpler patches, or even FX in pass-through.
Just to eliminate variables and find more information.

**Reply by jaffasplaffa** [reply]
The patch is very simple just a couple of envelopes and an oscillator, basically the start patch, with small edit.

I think it might be related to which track is armed for recording in Logic. 

Any armed track sounds right when I start playing. The not armed tracks doesn't sound right.

<img width="543" alt="Screenshot 2022-04-25 at 22 46 06" src="https://user-images.githubusercontent.com/14827192/165172244-d125d5a7-0154-4d26-9376-aafd4872b1d6.png">

**Reply by falkTX** [reply]
This is very weird, I wonder if it is AU specific...
But since Logic only supports AU, this will be hard to check.

Do you have any other host/DAW that supports AU that you can try?

**Reply by jaffasplaffa** [reply]
I don't have other DAWS that uses AUs, sorry.

Now that I think about it, I think we discussed this before, when Cardinal did not have AU, I tried using Patchwork, from Blue Cat Audio and loaded the VST version inside Patchwork, which acts as a vst>AU wrapper. I think I had the same result. I guess it is kind of the same as Logic loads Patchwork as an AU.

I will check it again tomorrow with Patchwork.
